packages/devtools_app/lib/src/timeline/timeline_processor.dart

```diff
@@ -661,6 +663,12 @@ class FullTimelineProcessor extends TimelineProcessor {
           .forEach(timelineController.recordTrace);
 
     for (var eventWrapper in _traceEvents) {
+      // This is a duplicate trace event. Skip it.
```

Comment:
is there a bug filed to track why there are dupes?

Reply:
added throughout the code referencing dupes